content/ja/docs/faq/_index.md

```diff
@@ -0,0 +1,551 @@
+---
+title: "よくある質問"
+weight: 8
+aliases: [
```

Comment:
Hi, @shuuji3 - please remove this "aliases" line. It will do something you do not expect (send people who want English pages to Japanese pages). See https://github.com/helm/helm-www/blame/master/content/en/docs/community/localization.md#L105 for more info.

Reply:
Sorry, I missed the instruction. I've fixed it!